projects/packages/forms/src/dashboard/integrations/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { __ } from '@wordpress/i18n';
+import './style.scss';
+
+const Integrations = () => (
+	<div className="jp-forms__integrations">
+		<div className="section-main">
+			<h3>{ __( 'Third party integrations coming soon…', 'jetpack-forms' ) }</h3>
```

Comment:
Since this string will never be displayed without the feature flag, it does not need to be translated.

```suggestion
			<h3>Third party integrations coming soon…</h3>
```

Reply:
@dhasilva - you mentioned this on slack, but I think this would be flagged in the linter. I'll also be updating this with actual code right away, so I think we just roll with it?

Reply:
Agree, the linter is tough on the PR checks, better translated than not

Reply:
Yep, let's go ahead